### DISCUSSION TITLE
Create Game Page - Remember previous Rules/Map/Size settings

### DISCUSSION THREAD

**Discussion by BoldBigflank** [BattlesnakeOfficial/feedback#215] [Feature Requests]
I tend to be doing the same game type over and over when I create games. It would be nice if my choices were saved in localstorage and were set when I choose to create a new game.

**Comment by BoldBigflank** [comment]
Related to this, the Create Rematch button could have a smaller, similar button which sends you back to the create page but is pre filled with that game's settings and snakes.

**Comment by chris-bsnake** [comment]
This is an interesting idea. We are looking to redesign and rebuild the Create Game form in the future and we can look at supporting some localstorage functionality when we implement that redesign
